design/api.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 // API the function to define the top-level API DSL of the application.
-var _ = a.API("fabric8-env", func() {
+var _ = a.API("env", func() {
```

Comment:
`goagen client` uses value of this to created client_dir.  With value `fabric8-env` it creates client dir `./tool/fabric8-env-cli` .. and now with `env` the dir is `./tool/env-cli`.